support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -170,4 +172,14 @@ public String getId() throws AbortException {
         }
     }
 
+    @Whitelisted
+    public @Nonnull String getAbsoluteUrl() throws AbortException {
+        String root = Jenkins.getActiveInstance().getRootUrl();
+        if (root == null) {
+            throw new AbortException("Jenkins root url is not configured!");
+        } else {
+            return root + build().getUrl();
```

Comment:
Right, it is only deprecated as a way of getting people’s attention so they do not use it in HTML pages, which is a different use case.